scipy/special/_ufuncs.pyx

```diff
@@ -6538,7 +6538,11 @@ cdef char ufunc_gamma_types[8]
 cdef char *ufunc_gamma_doc = (
     "gamma(z)\n"
     "\n"
-    "Gamma function.\n"
+    "Gamma function\n"
+    "\n"
+    ".. math::\n"
+    "\n"
+    "    \Gamma(z) = \int_0^\infty x^{z-1} e^{-x} dx = (z - 1)!\n"
```

Comment:
oh ok. thnx for the tip. i noticed the duplication but didn't know that there a script that does that.